### ISSUE TITLE
ValueError: Duplicated timeseries in CollectorRegistry

### ISSUE THREAD

**Issue by anxstj** [ansible/awx#13301]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Bug Summary

While upgrading my AWX system from 21.5.0 to 21.10.0 the following error occurred:

```
tools_awx_1 | ValueError: Duplicated timeseries in CollectorRegistry: {'awx01_example_com_messages_received'}
tools_awx_1 | 2022-12-07 16:21:09,927 INFO exited: wsbroadcast (exit status 1; not expected)
tools_awx_1 | 2022-12-07 16:21:09,927 INFO exited: wsbroadcast (exit status 1; not expected)
```

It seems that the wsbroadcast process is dying with this error and finally the tools_awx_1 container dies as well.

Changing the required *prometheus-client* back to 0.7.1 fixes (or workarounds) this error.

```
--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -253,7 +253,7 @@ pexpect==4.7.0
     #   ansible-runner
 pkgconfig==1.5.5
     # via -r /awx_devel/requirements/requirements.in
-prometheus-client==0.15.0
+prometheus-client==0.7.1
     # via -r /awx_devel/requirements/requirements.in
 psutil==5.9.4
     # via -r /awx_devel/requirements/requirements.in
```

This might be related to: https://github.com/prometheus/client_python/issues/815

### AWX version

21.10.0

### Select the relevant components

- [ ] UI
- [X] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [ ] Other

### Installation method

docker development environment

### Modifications

no

### Ansible version

2.12.2

### Operating system

Debian 11

### Web browser

Firefox

### Steps to reproduce

I have a setup with two controller nodes, awx01.example.com and awx02.example.com. The error message mentions always the other node. To reproduce the problem at least two controller nodes are needed, I guess.

### Expected results

The Prometheus client is registering the nodes properly.

### Actual results

```
tools_awx_1 | Adding {'awx01.example.com'} to websocket broadcast list
tools_awx_1 | Traceback (most recent call last):
tools_awx_1 |   File "/usr/local/bin/awx-manage", line 8, in <module>
tools_awx_1 |     sys.exit(manage())
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/__init__.py", line 201, in manage
tools_awx_1 |     execute_from_command_line(sys.argv)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
tools_awx_1 |     utility.execute()
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
tools_awx_1 |     self.fetch_command(subcommand).run_from_argv(self.argv)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
tools_awx_1 |     self.execute(*args, **cmd_options)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/django/core/management/base.py", line 398, in execute
tools_awx_1 |     output = self.handle(*args, **options)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/management/commands/run_wsbroadcast.py", line 170, in handle
tools_awx_1 |     loop.run_until_complete(task)
tools_awx_1 |   File "/usr/lib64/python3.9/asyncio/base_events.py", line 647, in run_until_complete
tools_awx_1 |     return future.result()
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/wsbroadcast.py", line 198, in run_per_host_websocket
tools_awx_1 |     stats = self.stats_mgr.new_remote_host_stats(h)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/analytics/broadcast_websocket.py", line 76, in new_remote_host_stats
tools_awx_1 |     self._stats[remote_hostname] = BroadcastWebsocketStats(self._local_hostname, remote_hostname)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/analytics/broadcast_websocket.py", line 124, in __init__
tools_awx_1 |     self._messages_received = Gauge(
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/prometheus_client/metrics.py", line 365, in __init__
tools_awx_1 |     super().__init__(
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/prometheus_client/metrics.py", line 143, in __init__
tools_awx_1 |     registry.register(self)
tools_awx_1 |   File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/prometheus_client/registry.py", line 43, in register
tools_awx_1 |     raise ValueError(
tools_awx_1 | ValueError: Duplicated timeseries in CollectorRegistry: {'awx01_example_com_messages_received'}
```

**Comment by relrod** [comment]
bisected to upstream prometheus/client_python@5c6af2d57da84d4b0d77b737f026fddd28ef2e11 which makes sense because we create a `_total` counter with the same prefix right above the line that is throwing, and this is no longer allowed.

**Comment by anxstj** [comment]
I can confirm that this error is fixed with 21.10.1. I will close this issue.
